src/phone_sms_terms.js

```diff
@@ -74,8 +74,14 @@ $('input[name=phone],input[name=mobile_phone]')
 //onSUBMIT
 $('form[name=act]').on('actionkitbeforevalidation', function() {
   if (window.console) {console.log('actionkitbefore validation! (mobile check)');}
-  var mobile = $('input[name=phone],input[name=mobile_phone]').val().replace(/\D/g, '');
+  var mobileFields = ('input[name=phone],input[name=mobile_phone]');
```

Comment:
should this start with a '$'?  i.e. `$('input[name=phone],input[name=mobile_phone]')`

You can probably just short circuit this all to something like:
```
var mobile = ($('input[name=phone],input[name=mobile_phone]').val()||'').replace(/\D/g, '');
```
but I understand and support your caution.